services/transaction_record.proto

```diff
@@ -120,6 +120,9 @@ message TransactionRecord {
     /**
      * In the record of an internal CryptoCreate transaction triggered by a user 
      * transaction with a (previously unused) alias, the new account's alias. 
+     * 
+     * In the record of an internal ContractCreate transaction triggered by a user
```

Comment:
Should we support the non-entity ID solidity address on a user-submitted contract create as well?

Reply:
You mean, populate this `alias` field with the Solidity address of a "top-level" contract?

Reply:
Yes, but also allow it as input to the contract create transaction body. Not saying we have to do that here but keep the design open for such future use.